types/recharts/index.d.ts

```diff
@@ -934,6 +936,8 @@ export interface YAxisProps extends EventAttributes {
     tickLine?: boolean | object;
     minTickGap?: number;
     tickSize?: number;
+    // The margin between tick line and the label
+    tickMargin?: number;
```

Comment:
Maybe there should be an `AxisProps` interface, since these two have a lot in common?